tests/scale/virt-api.go

```diff
@@ -115,11 +99,15 @@ var _ = Describe("[sig-compute] virt-api scaling", decorators.SigCompute, func()
 				},
 			},
 		}
-		oldcss := setccs(ccs)
-		DeferCleanup(restorescc, oldcss)
+		err := setccs(ccs, originalKv.Namespace, originalKv.Name)
+		Expect(err).ToNot(HaveOccurred())
+		DeferCleanup(func() {
+			err := setccs(originalKv.Spec.CustomizeComponents, originalKv.Namespace, originalKv.Name)
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
I think this function call happens after the normal execution of the test, right? What happens if this expect gets triggered?

Reply:
`DeferCleanup` will ignore all return values except for the last one.  If this last return value is a non-nil error `DeferCleanup` will fail the spec).

based on that i believe it's better to use:

`DeferCleanup(setccs, originalKv.Spec.CustomizeComponents, originalKv.Namespace, originalKv.Name)`